app/modules/elasticsearch/tasks.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from flask import current_app
+from gumby.models import Individual
+from sqlalchemy import create_engine, text
+from sqlalchemy.engine import Engine
+
+from app.extensions.celery import celery
+
+log = logging.getLogger(__name__)
+
+
+@celery.on_after_configure.connect
+def setup_periodic_tasks(sender, **kwargs):
+    sender.add_periodic_task(
+        10.0 * 60, load_indexes.s(), name='Load Elasticsearch Indexes'
+    )
+
+
+def create_wildbook_engine() -> Engine:
+    """Creates a SQLAlchemy Engine for connecting to the Wildbook database"""
+    return create_engine(current_app.config['WILDBOOK_DB_URI'])
+
+
+@celery.task
+def load_indexes():
+    wb_engine = create_wildbook_engine()
+    with wb_engine.connect() as wb_conn:
+        # Query for marked individual records
+        results = wb_conn.execute(text(WILDBOOK_MARKEDINDIVIDUAL_SQL_QUERY))
+        for result in results:
+            # Create the document object
+            indv = Individual(**result)
+            # Assign the elasticsearch document identify
+            indv.meta.id = f'markedindividual_{indv.id}'
+            # Augment records with houston data points
+            pass
+            # Save document to elasticsearch
+            indv.save(using=current_app.elasticsearch)
+
+
+WILDBOOK_MARKEDINDIVIDUAL_SQL_QUERY = """\
```

Comment:
this is one heck of a sql statement.  kudos.  i feel like i followed about 90% of it, and learned from the other 10% !